### DISCUSSION TITLE
How would I add a verification logo to specified accounts?

### DISCUSSION THREAD

**Discussion by aarnavpbhat** [pocketbase/pocketbase#1667] [Q&A]
I currently want to use this bootstrap icon <i class="bi bi-patch-check-fill"></i> right next to specific accounts, would there be some way to use Pocketbase's verification tool or another way to do so?

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand what do you mean.

You can check the users verification status by accessing the `verified` field, if that's what you are asking.

If you are looking from where to get the green check icon used in the Admin UI - it is an icon font from https://remixicon.com/.

**Reply by aarnavpbhat** [reply]
Is there a way to add customize how a username appears once the field is marked as verified?

**Reply by ganigeorgiev** [reply]
I'm sorry, I still don't understand what you are asking and what you are trying to do.

Users can update their username (_in the future there will be an option to prevent username reuse in case you don't want this_), but you can't change the visualization in the Admin UI.

**Reply by aarnavpbhat** [reply]
I'm still sorry for any confusion, I was wondering if there would be an option for me to add a verification icon next to usernames, like how twitter has one, simply by using pocketbase's verification tool. If not, are there any other ways?

**Reply by ganigeorgiev** [reply]
You can add your own "verified/checked" `bool` or any other field to the `users` collection.
For general overview about the collections, you can check https://pocketbase.io/docs/collections/.

**Reply by aarnavpbhat** [reply]
Once I verify a user, is there a possibility for how the user is viewed to change? Perhaps like an image or change in color that can be automatically applied?

**Reply by ganigeorgiev** [reply]
The admin UI visualization cannot change.

In your app you can do anything with the field value, including adding an icon, using some custom css class, color, etc.

**Reply by vigneshksaithal** [reply]
@aarnavpbhat you can trigger a serverless function to update image upon user verification.

**Comment by aarnavpbhat** [comment]
👍 Thanks for both of your help
